lib/Common/Memory/CustomHeap.h

```diff
@@ -89,7 +89,8 @@ struct Allocation
         } largeObjectAllocation;
     };
 
-    __field_bcount(size) char* address;
```

Comment:
Agree, was thinking along those lines.  Thanks for the helpful pointers! I think I prefer the latter solution, I was having a hard time spotting where in the existing there may be an `#ifdef` somewhere that keeps these from being defined. Will take a closer look.